### PR TITLE
#161254619 Implement social login

### PR DESCRIPTION
**Pull request by evamaina**
#### What does this PR do?
```This PR implements social login with Google and Facebook ```
#### Description of Task to be completed?
```Allow authentication with Facebook ```
```Allow authentication with Google ```
#### How should this be manually tested?
- Clone this repository
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
```cd ah-orcas-frontend```
- Checkout to the appropriate branch
 ```git checkout ft-social-login-161254619```
- Install the required dependencies and run the app:
```npm start```
- Open your browser and navigate to (http://localhost:3000/login)
- Choose the social media account you want to login in with
```Google``` or ```Facebook```

#### Any background context you want to provide?
```successful login should redirect you to the home page ```
#### What are the relevant pivotal tracker stories?
[#161254619](https://www.pivotaltracker.com/story/show/161254619)
#### Screenshots 
image
image
image

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

File: src/components/LoginPage/App.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+.fb-button {
+    background-color: #3B5998;
+}
```
[the remark this answers is not in the document]
Comment (abadojack) [reply src/components/LoginPage/App.scss:5]: This issue has been fixed (added empty line before and after rule declaration) yet it still shows an error here

**Review by abadojack — APPROVED** [review]
LGTM

**Review by TheSteelGuy — APPROVED** [review]
LGTM

**Review by wrotich — APPROVED** [review]
LGTM

**Pull request merged**